### PR TITLE
Remove velocity controllers for elmo devices

### PR DESCRIPTION
**Pull request by fmessmer**
includes #368 and #369 

removes velocity controllers for elmo devices as velocity mode is dangerous because joint limits are not considered correctly!

### CODE REVIEW

File: cob_default_robot_config/cob4-1/command_gui_buttons.yaml [hunk 1]
```diff
@@ -4,45 +4,13 @@
 #  buttons: [[button_name1,function_name1,parameter_name2,
 #    [button_name2,function_name2,parameter_name2]]
 
-group1:
+group10:
```
Comment (benmaidel) [inline cob_default_robot_config/cob4-1/command_gui_buttons.yaml:7]: why do you change the groups this drastically? is there a system behind?

Comment (fmessmer) [inline cob_default_robot_config/cob4-1/command_gui_buttons.yaml:7]: For cob4-2, there are more than 10 columns for knoeppkes.
According to @ipa-nhg, the ordering was faulty when using single-digit groups (0-9) together with dual-digit groups (10-99)...
The values have been copied from the current cob4-2 config...

Comment (benmaidel) [inline cob_default_robot_config/cob4-1/command_gui_buttons.yaml:7]: than this seems to be a bug on android gui side and/or cob_command_tools side.

Comment (floweisshardt) [inline cob_default_robot_config/cob4-1/command_gui_buttons.yaml:7]: in the command_gui the list is just sorted alphabetically, which means the following order

group1
group11
group1000000043
group2
group3

Switching to two-digit groups is OK for me

**Pull request merged**